src/msg/async/dpdk/DPDKStack.h

```diff
@@ -254,12 +254,11 @@ class DPDKStack : public NetworkStack {
   }
 
  public:
-  explicit DPDKStack(CephContext *cct): NetworkStack(cct) {
-    funcs.resize(cct->_conf->ms_async_max_op_threads);
```

Comment:
@tchaikov after this pr, `ms_async_max_op_threads` should be not used anymore, so maybe we could add a commit to get rid of it :)

```
$ grep -r ms_async_max_op_threads
doc/rados/configuration/ms-ref.rst:``ms_async_max_op_threads``
src/common/options.cc:    Option("ms_async_max_op_threads", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
src/common/legacy_config_opts.h:OPTION(ms_async_max_op_threads, OPT_U64)        // max number of worker processing threads for async messenger
src/msg/async/dpdk/DPDKStack.h:    funcs.resize(cct->_conf->ms_async_max_op_threads);
```

Reply:
@runsisi thanks for pointing this out. added and repushed.